src/styles/index.css

```diff
@@ -0,0 +1,91 @@
+@import "./stars.css";
+
+@import "./variables.css";
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
move this to `*`

Reply:
Hi! but it's properties of .card, is't it? When I move to * everything breaks